jenkins/src/main/fabric8/jenkins-deployment.yml

```diff
@@ -107,6 +107,12 @@ spec:
         - mountPath: "/home/jenkins/.docker"
           name: "jenkins-docker-cfg"
           readOnly: false
+        - mountPath: /var/jenkins_home/jobs
+          name: jenkins-jobs
+          readOnly: false
+        - mountPath: /var/jenkins_home/workspace
+          name: jenkins-workspace
+          readOnly: false
       serviceAccountName: "jenkins"
       volumes:
       - hostPath:
```

Comment:
Doh sorry this is mounting the docker socket - which I thought we no longer needed with @iocanel's latest updates, at least on OpenShift?